tests/plugins/widget/manual/dragwidget.html

```diff
@@ -0,0 +1,7 @@
+<div id="editor">
```

Comment:
I'm missing unit test for this case. It should be rather easy to write it: just fire `mousedown` event with appropriate data (remember about correct button values for IE8!) and check if `dragstart` event occurs.